lib/salus/package_utils/package_lock_json.rb

```diff
@@ -56,9 +56,11 @@ def record_dep_locations
 
     # recursively store all dependency names as keys
     def get_dep_names(data)
-      data['dependencies'].each do |name, dep_info|
-        @deps[name] = {}
-        get_dep_names(dep_info) if dep_info['dependencies']
+      if data.key?("dependencies")
```

Comment:
return unless data.key?("dependencies")
might be closer to the ruby style guide 


Bigger ask though is can we add a spec for the behavior?

Reply:
Yup adding